db/seeds.rb.example

```diff
@@ -0,0 +1,20 @@
+# Replace <your gmail name> with your gmail name below.
+
+Community.create!([
+  {name: "My Community", abbrv: "MC"}
+])
+
+Household.create!([
+  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
+])
+
+User.create!([
+  {email: "<your gmail name>@gmail.com", google_email: "<your gmail name>@gmail.com", first_name: "Alice", last_name: "Admin", mobile_phone: "17345551212", household_id: 1, admin: true}
+])
+
+Resource.create!([
+  {name: "Dining Room", community_id: 1, default_calendar_view: "week", meal_abbrv: "CH", hidden: false}
+])
+
+
+
```

Comment:
I just looked at 5 randomly chosen files and they all have a trailing
newline. Not sure if it's your editor.

"That is, in the editor, the lowest place I can place a cursor is at the
end of the last line that has content. Is that what you intend?", no that's
the opposite. If you do it that way, GitHub will show a 'no newline at end
of file' symbol.

On Fri, Sep 9, 2016 at 3:30 PM, Ken Winter <notifications@github.com> wrote:

> In db/seeds.rb.example
> <https://github.com/touchstone-cohousing/mess/pull/1#discussion_r78235879>
> :
>
> > +])
> > +
> > +Household.create!([
> > +  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
> > +])
> > +
> > +User.create!([
> > +  {email: "<your gmail name>@gmail.com", google_email: "<your gmail name>@gmail.com", first_name: "Alice", last_name: "Admin", mobile_phone: "17345551212", household_id: 1, admin: true}
> > +])
> > +
> > +Resource.create!([
> > +  {name: "Dining Room", community_id: 1, default_calendar_view: "week", meal_abbrv: "CH", hidden: false}
> > +])
> > +
> > +
> > +
>
> The MESS code files I see don't have a final newline (which would produce
> one trailing blank line). That is, in the editor, the lowest place I can
> place a cursor is at the end of the last line that has content. Is that
> what you intend?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/touchstone-cohousing/mess/pull/1/files/d49ab87ec9f05d1213b0a4b6bf614c19ebd67c5b#r78235879>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAq3cJkmqb-UI6-EXg7B5M-MTs41pyq_ks5qobPmgaJpZM4JzHfg>
> .
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech